SPECS/kata-containers/runtime-clh-Use-the-new-API-to-boot-with-TDX-firmware-td-shim.patch

```diff
@@ -1,35 +0,0 @@
-From 067e2b1e33b34b6bd0a55246f8f3db36dce3b3f2 Mon Sep 17 00:00:00 2001
```

Comment:
https://github.com/kata-containers/kata-containers/commit/067e2b1e33b34b6bd0a55246f8f3db36dce3b3f2 merged in 3.1.0 
<img width="445" alt="image" src="https://user-images.githubusercontent.com/30801614/234128469-5a037f76-edc1-4fe3-bb16-575fab5617ee.png">